tools/crowdin/actions/pr-creator/index.js

```diff
@@ -0,0 +1,85 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
```

Comment:
```suggestion
const githubRoot = require('@actions/github');
```

Renaming this so I can use `github` in a later suggestion (less stuff to rename that way)